docs/api.rst

```diff
@@ -13,7 +13,7 @@ Unless you have any special needs, all you need to know is:
   >>> ph = PasswordHasher()
   >>> hash = ph.hash("s3kr3tp4ssw0rd")
   >>> hash  # doctest: +SKIP
-  '$argon2i$v=19$m=512,t=2,p=2$5VtWOO3cGWYQHEMaYGbsfQ$AcmqasQgW/wI6wAHAMk4aQ'
+  '$argon2id$v=19$m=512,t=2,p=2$Wd3CJItnL93pSLG/Mvel2g$DnsqcrS7+Enwu8EdJrUX2Q'
```

Comment:
sanity check: I'm assuming the contract only ever was "pass me a string back that I once gave you to begin with"?

Reply:
There's not contract here really because the salt is random. So I basically created one by hand and put it there.